python/amici/petab_import.py

```diff
@@ -220,7 +220,7 @@ def import_petab_problem(
         model_output_dir: str = None,
         model_name: str = None,
         force_compile: bool = False,
-        **kwargs) -> amici.Model:
+        **kwargs) -> 'amici.Model':
```

Comment:
remove apostrophes?

Reply:
They were on purpose. This module is in principle usable without the SWIG interface. If we remove the apostrophes, it is not.

Reply:
Whats the practical usefulness of typehints without type?

Reply:
It's still visible to the implementer. And I thought that most tools would still be able to check.
I still see more value in keeping it like that, then removing it completely or making this module unusable for non-swig installations.

Reply:
okay :)